mitodl/open-discussions
# Added UI to upvote or downvote a comment

#### What are the relevant tickets?
Fixes #84

#### What's this PR do?
Adds UI to upvote or downvote a comment

#### How should this be manually tested?
Click the up or down button next to the comment. If you click a button and it's already highlighted, it should stop being highlighted.

#### Any background context you want to provide?
I'm not sure why the score doesn't change. We should look into that at some point with more than one user
